plugins/coral-plugin-love/server/typeDefs.graphql

```diff
@@ -0,0 +1,70 @@
+enum ACTION_TYPE {
+
+    # Represents a Love.
+    LOVE
```

Comment:
The strategy that I've been recommending is to use the name of the plugin in all globally scoped places. In this case, this plugin works with an action type of LOVE. If another plugin also uses LOVE as an action type there may be unexpected consequences. 

To be super strict, we could insist that this kind of thing is written 
```
enum ACTION_TYPE {

  CORAL-PLUGIN-LOVE
}
```

This is more verbose and gets ugly when writing graphql types but does self-document all the way to the database level as the strings in the action show that it came from a plugin.